dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureUtilities.java

```diff
@@ -115,73 +156,180 @@ public static Set<String> allTestFeatures() {
     }
 
     /**
-     * Get the set of public feature short names by reading the feature files from a liberty install
+     * Tell if a feature a public, non-test feature.
      *
-     * @param  libertyInstallRoot the wlp directory containing the liberty install
+     * @param  featureFile A feature file.
      * @return                    the list of public short names
      */
-    public static Set<String> getFeaturesFromServer(File libertyInstallRoot, boolean openLibertyOnly) {
+    public static boolean isPublicFeature(File featureFile) {
+        String name = featureFile.getName();
+        return (name.startsWith("io.openliberty.") || name.startsWith("com.ibm."));
+    }
+
+    /**
+     * Tell if a feature is a versionless feature.
+     *
+     * Versionless feature names do not have a '-' character.
+     *
+     * @param  feature A feature name or short name.
+     *
+     * @return         True or false telling if the feature is a versionless
+     *                 feature.
+     */
+    public static boolean isVersionless(String feature) {
+        return (feature.indexOf('-') == -1);
```

Comment:
ideally, we want to use the io.openliberty.versionless prefix here, but it seems we are only getting the short name.